Makefile

```diff
@@ -139,4 +139,4 @@ nxdl: $(NXDL_APPDEF) $(NXDL_CONTRIB) $(NXDL_BC)
 # License along with this library; if not, write to the Free Software
 # Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
 #
-# For further information, see http://www.nexusformat.org
+# For further information, see http://www.nexusformat.org
```

Comment:
This change should be removed.

Reply:
There is no changes. probably fairmat branch has one or more spaces at the end of line which is not present in the current branch.

Reply:
Yes, I think there was a newline added at the end of the file. Could you remove it? Then we don't have it in the diff when we merge

Reply:
No, there are no new line, it is showing changes on line-142, probably there are some spaces at the end of line in `fairmat` branch, that is showing the difference on line 142.

Reply:
In fairmat is a newline which was removed here. I readded it.